vite.content.config.ts

```diff
@@ -0,0 +1,33 @@
+import { resolve } from 'path'
+import { defineConfig } from 'vite'
+import getPagesViteConfig from './vite.config'
+
+const isDev = process.env.NODE_ENV !== 'production'
+const r = (...args: string[]) => resolve(__dirname, ...args)
```

Comment:
These can be imported from `scripts/utils.ts`

Reply:
Yep , fixed